packages/worker/src/worker.ts

```diff
@@ -598,13 +598,29 @@ export class Worker {
                         isReplaying: activation.isReplaying,
                       };
 
+                      const patchJobs = activation.jobs.filter(
+                        (
+                          j
+                        ): j is NonNullableObject<
+                          Pick<coresdk.workflow_activation.IWFActivationJob, 'notifyHasPatch'>
+                        > => j.notifyHasPatch != null
```

Comment:
Oh my god this is some nauseating formatting (I know it's auto-formatted, but, jesus)

Reply:
lol, I can declare the type somewhere else so the formatting is fixed